3.3/test/run

```diff
@@ -84,7 +89,8 @@ wait_for_cid() {
   local sleep_time=1
   local attempt=1
   local result=1
-  info "Waiting for application container to start"
+  local cargs=${CONTAINER_ARGS:-}
+  info "Waiting for application container to start $cargs"
```

Comment:
Same question then, why is it needed there?! How about `docker run $CONTAINER_ARGS --rm --cidfile=$cid_file -p $test_port ${IMAGE_NAME}-testapp`?